### DISCUSSION TITLE
Xdrip disable Vibration

### DISCUSSION THREAD

**Discussion by 3DRZ** [NightscoutFoundation/xDrip#2968] [General]
I use Xdrip with my Libre2. This works fine but when it looses connection Xdrip goes totally crazy and vibrates the whole time.
I searched for settings but could not find something.
The problem is, that it vibrates such in an annoying manner and amount that I'm going crazy with Xdrip.
I mean it would be fine when it vibrates maybe 1 time and then tries to reconnect but vibrating a few times every minute until it reconnects 5 minuts later makes no sense.
Is this a known behavior or a bug or semething?

**Comment by Navid200** [comment]
xDrip has different alerts.  If you look into the logs, you can tell which alert it is.
It could be the missed reading alert.
Also, you can disable vibration for each alert.

https://navid200.github.io/xDrip/docs/Alerts_page.html

The missed reading alert allows you to control when it should repeat if you don't snooze it and when it should repeat after a snooze.

**Reply by 3DRZ** [reply]
Thats the problem: I have disabled the missed reading alert. And it keeps alerting anyway

**Reply by Navid200** [reply]
Look at the logs to see what the alert is for.

**Comment by 3DRZ** [comment]
![Screenshot_2023-07-26-16-06-39-11_2dd2c1bcb8f659cd696bcba2072f751b.jpg](https://github.com/NightscoutFoundation/xDrip/assets/34788659/ab16a0e1-d311-4c34-8fc9-699f8efb1f6f)

I think it is a disconnect

**Reply by Navid200** [reply]
You can scroll through the logs to go back and see the logs when you got the alert.
xDrip never issues an alert without creating a corresponding log.

After you find the log, we can see what alert it is.

**Reply by 3DRZ** [reply]
This is the log of the exact moment of the alert.

![Screenshot_2023-07-26-18-44-27-88_2dd2c1bcb8f659cd696bcba2072f751b.jpg](https://github.com/NightscoutFoundation/xDrip/assets/34788659/d7abc56a-7b53-4042-9676-8613c7f6f87f)

But I cannot see any alert message in the logs.
I can also.search throug logs. For what should I look? The term "alert" is not in there.

**Reply by Navid200** [reply]
How do you know it is xDrip that is making the noise?
Is it possible that it is another app that does it?

**Reply by 3DRZ** [reply]
100% xdrip cause it vibrates 1 time long then 1 time shorter. This happens every time it looses connection to the libre. Then it goes like crazy and keeps vibrating until it gets the connection back. Sometimes it vibrates like 2 times in a minut but sometimes also a few times in a row.
Seems like xdrip tries to get the connection back and when its not able to get the connection it vibrates

**Reply by Navid200** [reply]
How do you know it is xDrip?
xDrip does not alert without creating a log.

**Reply by 3DRZ** [reply]
Thats pretty easy:
- it always happens when connection in xdrip is lost (it never happens as long it gets the readings every 5 minutes)
- it never happens when i disable xdrip (for example when i change sensor)
- when it happens I look at my phone and think there was a massage but then theres only xdrip on the notification screen
- it vibrates in a pattern (two times within 3 seconds or so) wich no other app does

So there is no possibility I could imagine that makes me think that its not xdrip.
+ I have read a few days ago from a guy with similar problem:
https://www.reddit.com/r/diabetes_t1/comments/12fhq2e/what_is_causing_my_xdrip_to_constantly_beep_when/

**Comment by Navid200** [comment]
Please post a screenshot of the classic status page:
https://navid200.github.io/xDrip/docs/xDrip-Version.html

**Reply by 3DRZ** [reply]
![Screenshot_2023-07-26-20-24-38-09_2dd2c1bcb8f659cd696bcba2072f751b.jpg](https://github.com/NightscoutFoundation/xDrip/assets/34788659/c2094b32-bb6d-4c4f-b326-21821876cbcc)

**Reply by Navid200** [reply]
Will you be able to record a clip so that we can see and hear?

**Reply by 3DRZ** [reply]
I tried but you cant really hear the vibration. 
But as soon as it goes over the 5 minutes 2 vibration come up.

Thats how it behaves.

Heres the video, vibration is 0,5 sec after video starts:https://drive.google.com/file/d/166XSz2Nm3SwNNjU3O8ztkNFcv7reLO_b/view?usp=drivesdk
